src/app/tic-tac-toe.component.ts

```diff
@@ -61,7 +61,7 @@ export class TicTacToeComponent {
     this.cells = Array(this._size).fill([]);
     for (let i = 0; i < this._size; i++) {
       this.cells[i] = Array(this._size).fill(Cell.Clear);
-    }
+    } // review
```

Comment:
1. Take a look at TypeScript's *for-of* loops.
2. Screw loops and write this in a pretty, functional way by simply putting the second Array.fill inside of the first one.